src/_scss/components/tooltips/_customTooltipContent.scss

```diff
@@ -83,7 +83,11 @@
 
     h2 {
         text-align: center;
-        font-size: $base-font-size;
+        font-size: rem(16);
```

Comment:
This file is shared styling for all of our tooltips of this type, and will have an affect across the site. Not sure if that is intentional. We can use a more specific class name if you only want this to affect the homepage COVID tooltip.

Reply:
Disregard! This is nested in a class for the homepage 👍